src/styles/blocks/card.scss

```diff
@@ -0,0 +1,137 @@
+.container {
+  display: grid;
+  justify-content: center;
+  grid-gap: 46px 48px;
+  padding: 50px 40px;
+
+  @include number-of-grid-template-columns;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  font-family: inherit;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  box-sizing: border-box;
```

Comment:
put it to the `*` selector
```suggestion
```

Reply:
<img width="963" alt="Screenshot 2024-02-05 at 16 58 31" src="https://github.com/mate-academy/layout_catalog/assets/14590796/24755d46-ca64-43b5-9e30-b59ea5120246">

Reply:
> box-sizing: border-box;

`box-sizing: border-box;` it's the only case where you can put under `*` selector. But it is ok for this task to don't do it